packages/ui/src/components/PostScreenAuthorRow.tsx

```diff
@@ -0,0 +1,48 @@
+import type * as db from '@tloncorp/shared/dist/db';
+
+import { SizableText, XStack } from '../core';
+import { Avatar } from './Avatar';
+import ContactName from './ContactName';
+import { ListItem } from './ListItem';
+import Pressable from './Pressable';
+
+export default function PostScreenAuthorRow({
+  parentPost,
+  timeDisplay,
+  setShowComments,
+}: {
+  parentPost: db.Post;
+  timeDisplay: string;
+  setShowComments: (show: boolean) => void;
+}) {
+  return (
+    <Pressable onPress={() => setShowComments(true)}>
```

Comment:
Just checking -- we've got a few `AuthorRow`s now; anything abstractable?

Reply:
They all have some pretty unique layouts (detail view author row vs gallery view vs notebook view [hasn't been extracted to a component yet] vs chat view), so at the moment they seem like they're ok if they're separated imo.

Reply:
Actually, we could just pass some extra props and make one component.